unittest/unit/update-bcd.ts

```diff
@@ -861,5 +861,205 @@ describe('BCD updater', () => {
+            downstreamResult: false
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: '85'},
+              chrome_android: false
+            })
+          );
+        });
+
+        it('omitted from downstream test results', () => {
+          const actual = mirroringCase({
+            support: {
+              chrome: {version_added: '85'},
+              chrome_android: 'mirror'
+            },
+            downstreamResult: null
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: '85'},
+              chrome_android: 'mirror'
+            })
+          );
+        });
+      });
+
+      describe('supported upstream (with flags)', () => {
+        it('supported in downstream test results', () => {
+          const actual = mirroringCase({
+            support: {
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: 'mirror'
+            },
+            downstreamResult: true
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: [
+                {version_added: '86'},
+                {flags: [{}], version_added: '85'}
+              ]
+            })
+          );
+        });
+
+        it('unsupported in downstream test results', () => {
+          const actual = mirroringCase({
+            support: {
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: 'mirror'
+            },
+            downstreamResult: false
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: false
+            })
+          );
+        });
+
+        it('omitted from downstream test results', () => {
+          const actual = mirroringCase({
+            support: {
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: 'mirror'
+            },
+            downstreamResult: null
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: '85', flags: [{}]},
+              chrome_android: 'mirror'
+            })
+          );
+        });
+      });
+
+      describe('unsupported upstream', () => {
+        it('supported in downstream test results', () => {
+          const actual = mirroringCase({
+            support: {
+              chrome: {version_added: false},
+              chrome_android: 'mirror'
+            },
+            downstreamResult: true
+          });
+          assert.deepEqual(
+            actual,
+            bcdFromSupport({
+              chrome: {version_added: false},
+              chrome_android: {version_added: '86'}
```

Comment:
Again this is not a valid inference. The code should be adding a range here, right? (A human then needs to fix it, because the range isn't valid in BCD.)

Reply:
Yes! Thanks! In the `version_added` property in surrounding tests, I'm seeing ranges like `'0> ≤86'` and like `'≤86'`. [This project's description of range format](https://github.com/foolip/mdn-bcd-collector/blob/cec271d457776a375939a1b674c69fec154ea1d1/README.md#custom-ranged-version-format) doesn't seem to apply to the `0>` part. Based solely on a mathematical background, these seem equivalent. Is that right? If so, which should this test use?

Reply:
That range format is a bit odd and basically is to shout "human, look at me!"

But the idea is that if we've confirmed no support in release A, and having support in version C, but no results from version B in between them, then we generate `A> ≤C` to record the information that we had, instead of the simpler `≤C`. I guess there's just no simplification done when A is the first release, since we haven't run into that with current browsers.

If you use mock data that doesn't result in these ranges that'd be fine, but if you'd like to exercise it, just use whatever is currently produced, I don't think it's wrong, just hard to parse.

Reply:
Currently (as per the first commit on this branch), the script leaves the value as `"mirror"`. I'll just choose one for now and adjust if I find that the simplest fix favors the other.